smoke-tests/src/test/java/com/splunk/opentelemetry/helper/AbstractTestContainerManager.java

```diff
@@ -44,7 +44,6 @@ protected Map<String, String> getAgentEnvironment(
     // export metrics every 1s
     environment.put("SPLUNK_METRICS_EXPORT_INTERVAL", "1000");
```

Comment:
I removed it from that test class, but I don't think we can remove `SPLUNK_METRICS_ENABLED` as a feature because that's the only mechanism (that I know of) to disable metrics. Some users won't want them enabled by default.

Reply:
Ah I guess we just recommend `OTEL_METRICS_EXPORTER=none` instead. Way cleaner! 🥳

Reply:
`SPLUNK_METRICS_ENABLED` can not be used to disable metrics because it defaults to false. In 2.x this flag only controls splunk specific metrics. With the removal of thread pool metrics there are no more metrics that are enabled with this flag, profiling metrics don't depend on this flag.